models/intel/smartlab-object-detection-0001/accuracy-check.yml

```diff
@@ -0,0 +1,35 @@
+models:
+  - name: smartlab-object-detection-0001
+
+    launchers:
+      - framework: dlsdk
+        device: CPU
+        adapter:
+          type: yolox
+          anchors: yolox
+          classes: 10
+
+    datasets:
+      - name: DSI1867
```

Comment:
you should not mention internal dataset id in any public files, please gived something neutral

Reply:
changed all of DSI**** name